src/html/tree.ml

```diff
@@ -70,6 +68,34 @@ let page_creator ~config ~url name header toc content =
         Html.script ~a:[ Html.a_src highlight_js_uri ] (Html.txt "");
         Html.script (Html.txt "hljs.initHighlightingOnLoad();");
       ]
+    in
+    let meta_elements =
+      if uses_katex then
+        let katex_css_uri = file_uri theme_uri "katex.min.css" in
+        let katex_js_uri = file_uri support_uri "katex.min.js" in
+        default_meta_elements
+        @ [
+            Html.link ~rel:[ `Stylesheet ] ~href:katex_css_uri ();
+            Html.script ~a:[ Html.a_src katex_js_uri ] (Html.txt "");
+            Html.script
+              (Html.cdata_script
+                 {|
+          document.addEventListener("DOMContentLoaded", function () {
+            var elements = document.getElementsByClassName("odoc-katex-math");
```

Comment:
This doesn't work as expected because `document.getElementsByClassName()` returns a _live_ HTMLCollection (see [the warning](https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementsByClassName)).